niche_vlaanderen/niche.py

```diff
@@ -784,6 +784,12 @@ def plot(self, key, ax=None, fixed_scale=True):
         norm = None
         v = None
 
+        # Inform user that data is not available to plot
+        # (key is present, but value is None)
+        if key not in self._inputfiles and key in self._inputarray:
+            if not self._inputarray[key]:
+                raise KeyError(f"{key} data is not available to plot.")
```

Comment:
perhaps add (constant value).

Reply:
The situation covered here is when the dict value of these `self._inputarray[key]` are None.